include/priv/instr.h

```diff
@@ -144,8 +144,8 @@ typedef enum _InstrType {
     IT_CMOVS, IT_CMOVNS, IT_CMOVP, IT_CMOVNP,
     IT_CMOVL, IT_CMOVGE, IT_CMOVLE, IT_CMOVG,
 
-    IT_SETO, IT_SETNO, IT_SETC, IT_SETNC, IT_SETZ, IT_SETNZ, IT_SETBE, IT_SETA,
-    IT_SETS, IT_SETNS, IT_SETP, IT_SETNP, IT_SETL, IT_SETGE, IT_SETLE, IT_SETG,
+    IT_SETA, IT_SETAE, IT_SETB, IT_SETBE, IT_SETE, IT_SETG, IT_SETGE, IT_SETL,
```

Comment:
It seems like a bad idea to give meaning to the numeric values of enum symbols that aren't explicitly assigned. I would either a) accept some additional boilerplate for matching the conditional instructions or b) explicitly assign instruction opcodes as enum symbol values, e.g.  IT_SETA = 0x0f97.